### DISCUSSION TITLE
DNS resolution failed after portainer gui update

### DISCUSSION THREAD

**Discussion by thegameksk** [haugene/docker-transmission-openvpn#2425] [Q&A]
Not sure what happened but after portaier gui update upon restart my transmissionopenvpn container isnt working. Under logs it says 
WARNING: initial DNS resolution test failed
Creating TUN device /dev/net/tun
/etc/openvpn/start.sh: line 34:    11 Aborted                 (core dumped) nslookup ${HEALTH_CHECK_HOST:-"google.com"} > /dev/null 2>&1

and

curl: (6) getaddrinfo() thread failed to start.

Also under status it says unhealthy.

Nothing in my internet connection has changed. Thank you.
[_transvpn_logs.txt](https://github.com/haugene/docker-transmission-openvpn/files/10047131/_transvpn_logs.txt)

**Comment by CosmichunterX** [comment]
I'm seeing exactly the same errors after restarting Transmission.
/etc/openvpn/start.sh: line 34:    10 Aborted                 (core dumped) nslookup ${HEALTH_CHECK_HOST:-"google.com"} > /dev/null 2>&1

I've recreated the container with the latest image and tried changing the DNS but the issue remains the same.

**Reply by thegameksk** [reply]
> I'm seeing exactly the same errors after restarting Transmission. /etc/openvpn/start.sh: line 34: 10 Aborted (core dumped) nslookup ${HEALTH_CHECK_HOST:-"google.com"} > /dev/null 2>&1
> 
> I've recreated the container with the latest image and tried changing the DNS but the issue remains the same.

https://github.com/haugene/docker-transmission-openvpn/issues/2410
Here is a temporary fix until there is an update. You can revert to an older transvpn version.
